readme.md

```diff
@@ -1,7 +1,7 @@
 # Boilerplate for layout tasks
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_snake/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_snake/report/html_report/)
+- [DEMO LINK](https://11Olya112.github.io/layout_snake/)
```

Comment:
https://user-images.githubusercontent.com/38065883/174786542-c5c930de-9504-42ee-b646-ce5bc16f6514.mov

There should be no horizontal scroll